content/jobs/jobs.yaml

```diff
@@ -0,0 +1,14 @@
+- company: "NewHaven.IO"
+  jobURL: "https://newhaven.io"
+  jobTitle: "Community Moderator"
+  location: "New Haven, CT // Remote"
+  tech: "Slack, MeetUp, GatsbyJS"
+  description: "We're looking for help with some community moderation. As vaccines keep rolling out we're looking at ways we can begin to breathe life back into the group. We want to wade back into holding some events and we're going to need help keeping an eye our Slack team and Meetup page. Over time we can fold whoever takes this role into our organizing and leadership teams. If you've gotten some value from IO in the past we'd love your help in carrying it forward."
```

Comment:
Surely salary here is volunteer

Reply:
Good catch. I was checking a requirement filter and forgot to put it back.